test/fixtures/add/config/new.yaml

```diff
@@ -0,0 +1,5 @@
+subcomponents:
```

Comment:
Likewise,  the `TestSetValue` test actually sets this value, so we don't need it checked in.